src/controllers/user/official-contact-email-verification.ts

```diff
@@ -113,12 +113,13 @@ export const postOfficialContactEmailVerificationMiddleware = async (
 
     return next();
   } catch (error) {
+    const { organization_id } = req.params;
```

Comment:
this could throw an error if req.params is undefined (not sure if this can happen though).

Reply:
K better bypass the typescripts errors then ?